en-US/Docs/Loopback.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: docs
+title: Communicating with localhost (loopback)
+description: How to turn on Embedded Mode for Windows desktop and Windows Mobile editions
+keyword: windows runtime, windows iot, localhost, loopback
+permalink: /en-US/Docs/Loopback.htm
+lang: en-US
+---
+
+# Communicating with localhost (loopback)
+
+On **Windows IoT Core** If you want to create a TCP/IP connection between 2 processes running on the same device and one of them is a UWP app you must enable localhost loopback.
+
+## Loopback and the debugger 
+By default, running under the debugger enables outbound loopback automatically for that debug session only.  You shouldn’t have to do anything as long as the loopback checkbox is checked in the debugger settings for your startup project.  If you want to implement a socket listener the you must enable localhost loopback for inbound connections (see below).
+ 
+## Enabling the loopback policy
+The localhost loopback policy for **Windows IoT Core** is controlled by the following registry key:

```

Comment:
I would explicitly define that this is for inbound ... something like: 

The localhost policy for IoT enables inbound connections for UWP apps.  It requires that the following registry key be set: